tests/test_browser.py

```diff
@@ -4567,6 +4567,13 @@ def run(emcc_args=[]):
     run(['-s', 'ASSERTIONS=1'])
     run(['-s', 'PROXY_TO_PTHREAD=1'])
 
+  # Tests that time in a pthread is relative to the main thread, so measurements
+  # on different threads are still monotonic, as if checking a single central
+  # clock.
+  @requires_threads
+  def test_pthread_reltime(self):
+    self.btest(path_from_root('tests', 'pthread', 'test_pthread_reltime.cpp'), expected='3', args=['-s', 'USE_PTHREADS=1', '-s', 'PTHREAD_POOL_SIZE=1'])
```

Comment:
Can this in node now too?  We should probably have a plan for centralizing the thread tests that run in both browser and node.  Perhaps `test_threads.py`?

Reply:
Makes sense to me, but since it seems like a larger undertaking I hope this does not block landing this fix.